core/src/test/java/io/questdb/griffin/engine/functions/regex/LikeFunctionFactoryTest.java

```diff
@@ -278,4 +278,53 @@ public void testNotLikeStringMatch() throws Exception {
             }
         });
     }
+
+    @Test
+    public void testLikeEscapeOneSlashes() throws Exception {
+        String createTable = "CREATE TABLE myTable (name string)";
+        String insertRow = "INSERT INTO myTable  (name) VALUES ('The path is \\_ignore');";
+
+        String query = "SELECT * FROM myTable WHERE name LIKE 'The path is \\_ignore';";
+        String expected1 = "name\n";
+        String expected2 = "name\n";
+
+        assertQuery(expected1, query, createTable, null, insertRow, expected2, true, true, true, true);
+    }
+    @Test
+    public void testLikeEscapeTwoSlashes() throws Exception {
+        String createTable = "CREATE TABLE myTable (name string)";
+        String insertRow = "INSERT INTO myTable  (name) VALUES ('The path is \\_ignore');";
+
+        String query = "SELECT * FROM myTable WHERE name LIKE 'The path is \\\\_ignore';"; // two backslashes
+        String expected1 = "name\n";
+        String expected2 = "name\nThe path is \\_ignore\n";
+
+        assertQuery(expected1, query, createTable, null, insertRow, expected2, true, true, true, true);
+    }
+
+    @Test
+    public void testLikeEscapeThreeSlashes() throws Exception {
+        String createTable = "CREATE TABLE myTable (name string)";
+        String insertRow = "INSERT INTO myTable  (name) VALUES ('The path is \\_ignore');";
+
+        String query = "SELECT * FROM myTable WHERE name LIKE 'The path is \\\\\\_ignore';"; // three backslashes
+        String expected1 = "name\n";
+        String expected2 = "name\nThe path is \\_ignore\n";
+
+        assertQuery(expected1, query, createTable, null, insertRow, expected2, true, true, true, true);
+    }
+
```

Comment:
Comments are a bit misleading . 
'\\\\' string inserted in java test gets escaped by java first so \\ is passed to qdb.
Since there's no escaping on insert (yet) it turns into two backslashes . 

'\\\\\\\\' as like parameter first is sent to engine as '\\\\' and interpreted as \\ by logic you add in PR .